etc/kube/start-minikube-vm.sh

```diff
@@ -95,3 +95,7 @@ set -x
 
 # Kill pachctl port-forward and kubectl proxy
 killall kubectl || true
+
+# Port forward to etcd
+export ETCD_POD=$(kubectl describe -l app=etcd pod | awk 'NR == 1 {print $2}')
```

Comment:
You can actually get the pod name directly from kubectl using the `-o` option:
`kubectl get pod -l suite=pachyderm,app=etcd -o jsonpath={.items[].metadata.name}'